src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const accountanting = [];
```

Comment:
Interesting solution, but let's simplify it: 

Initialize this variable as number 
```suggestion
  const accountanting = 0;
```